File: Ej1extra.java
package ej1extra;

import javax.swing.JOptionPane;

class Ej1extra {

	public static void main(String[] args) {
		double ch, vh, sb, os, jub, sn;
		
		ch=Double.parseDouble(JOptionPane.showInputDialog("Ingrese cant horas"));
		vh=Double.parseDouble(JOptionPane.showInputDialog("Ingrese cant horas"));
		sb=ch*vh;
		os=sb*0.07;
		jub=sb*0.10;
		sn=sb-os-jub;
		JOptionPane.showMessageDialog(null, "***** Recibo de sueldo ****** \nsueldo bruto:"
		+ sb + "\ndescuento obra social: " + os + "\ndescuento jubilacion: " + jub
		+ "\nsueldo neto a cobrar: " + sn);
		


	}

}

File: Ej1extra.js
let ch = parseFloat(prompt("Ingrese cant horas"));
let vh = parseFloat(prompt("Ingrese valor por hora"));

let sb = ch * vh;
let os = sb * 0.07;
let jub = sb * 0.10;
let sn = sb - os - jub;

alert("***** Recibo de sueldo ****** \nSueldo bruto: " + sb + "\nDescuento obra social: " + os + "\nDescuento jubilación: " + jub + "\nSueldo neto a cobrar: " + sn);

